### DISCUSSION TITLE
How to run jobs with containers dependencies?

### DISCUSSION THREAD

**Discussion by thiagoolsilva** [dagger/dagger#4048] [Help]
I want to create two containers that communicate between them to run integration tests. is it possible to run this kind of situation using dagger.io? I've tried some approaches without any success. So, if anyone has any suggestions, please, let me know. I'd love to know how to solve this problem using dagger.io. 

Ex: Backend NodeJs (Docker A) --(communicate)--> Database MySQL (Docker B)

**Comment by jlongtine** [comment]
Short answer is that it isn't possible (or at least not at all straightforward) currently. 

The team has been thinking about this (and other) use cases for a while. 

We call them "services". 

You can keep an eye on the development:
- https://github.com/dagger/dagger/issues/3173
- https://github.com/dagger/dagger/issues/3037

And, a sense of how we might implement some of it here: https://github.com/dagger/dagger/issues/3850#issuecomment-1329899415

**Reply by thiagoolsilva** [reply]
Thanks @jlongtine
